ptbr_sampler/cli.py

```
@@ -0,0 +1,869 @@
+from pathlib import Path
+from typing import Any, Optional
+
+# Import uvloop for faster asyncio performance
+import uvloop
+
+# Shut down standard logging to prevent INFO logs from libraries
+import logging
+logging.getLogger().setLevel(logging.WARNING)
+for handler in logging.getLogger().handlers:
+    handler.setLevel(logging.WARNING)
+
```

Ruff (0.12.2)

Module level import not at top of file

(E402)